src/content/vrf/v2/subscription/supported-networks.mdx

```diff
@@ -136,6 +136,27 @@ Testnet LINK is available from https://faucets.chain.link/bnb-chain-testnet
 | Maximum Confirmations | 200                                                                                                                                                                                     |
 | Maximum Random Values | 500                                                                                                                                                                                     |
 
+### Polygon Amoy testnet
+
+<Vrf2_5Common callout="supportednetworks" />
+
+<Aside type="tip" title="Important">
+  The LINK provided by the [Polygon (Matic) Bridge](https://wallet.polygon.technology/polygon/bridge) is not ERC-677
+  compatible, so cannot be used with Chainlink oracles. However, it can be [**converted to the official LINK token on
+  Polygon (Matic) using Chainlink's PegSwap service**](https://pegswap.chain.link/)
+</Aside>
```

Comment:
```suggestion
```

LINK on Amoy is available from the faucet, so the bridge notice is not required for testnets. https://faucets.chain.link/polygon-amoy

It might technically apply to the Sepolia to Amoy bridge, but the notice is mostly for mainnet users who can't just get assets from a faucet.